### PR TITLE
Added loaded yum plugins count scenario

### PR DESCRIPTION
**Pull request by vijay8451**
- Fixed https://github.com/SatelliteQE/satellite6-upgrade/issues/288
- Test results:

> [root@vijsingh robottelo]# pytest tests/upgrades/ -v -k "test_pre_scenario_yum_plugins_count" -s
> =========================================================== warnings summary ===========================================================
> tests/upgrades/test_yum_plugins.py::Scenario_yum_plugins_count::test_pre_scenario_yum_plugins_count
>   /home/vijsingh/Satellite/robottelo/env/lib/python3.6/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
>     InsecureRequestWarning)
>   /home/vijsingh/Satellite/robottelo/env/lib/python3.6/site-packages/bugzilla/base.py:114: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
>     self.tokenfile = SafeConfigParser()
>   /home/vijsingh/Satellite/robottelo/env/lib/python3.6/site-packages/bugzilla/base.py:558: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
>     c = SafeConfigParser()
> 
> -- Docs: https://docs.pytest.org/en/latest/warnings.html
> ======================================== 1 passed, 29 deselected, 3 warnings in 825.44 seconds =========================================
> 
> 
>[root@vijsingh robottelo]# pytest tests/upgrades/ -v -k "test_post_scenario_yum_plugins_count" -s
> =========================================================== warnings summary ===========================================================
> tests/upgrades/test_yum_plugins.py::Scenario_yum_plugins_count::test_post_scenario_yum_plugins_count
>   /home/vijsingh/Satellite/robottelo/env/lib/python3.6/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
>     InsecureRequestWarning)
> 
> -- Docs: https://docs.pytest.org/en/latest/warnings.html
> ======================================== 1 passed, 29 deselected, 1 warnings in 208.3 seconds =========================================
>

### CODE REVIEW

File: tests/upgrades/test_yum_plugins.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        tools_repo, rhel_repo = self._create_custom_rhel_tools_repos(product)
+        product.sync()
+        for repo in (tools_repo, rhel_repo):
+            content_view.repository.append(repo)
+        content_view = content_view.update(['repository'])
+        content_view.publish()
+
+        self._install_or_update_package(client_container_id, "katello-agent", update=True)
+        self._run_goferd(client_container_id)
+        self._check_yum_plugins_count(client_container_id)
```
Comment (omkarkhatavkar) [inline tests/upgrades/test_yum_plugins.py]: What is the difference between _check_yum_plugins_count in pre and post-upgrade step? What do you think about adding the same step in pre-upgrade as well?

Comment (vijay8451) [reply tests/upgrades/test_yum_plugins.py]: @omkarkhatavkar yeah it only requires to check at upgraded(post upgrade) version to see such[1] type of issue if any in upgraded package/tool: 
[1] https://bugzilla.redhat.com/show_bug.cgi?id=1625649

Comment (omkarkhatavkar) [reply tests/upgrades/test_yum_plugins.py]: My intention was raising this, what if loading plugin issue itself exist in pre-upgrade. There is no way that you could tell that it was working in pre-upgrade.

Comment (vijay8451) [reply tests/upgrades/test_yum_plugins.py]: @omkarkhatavkar this time bug is fixed for 6.5.

**Pull request merged**